README.md

```diff
@@ -51,6 +51,17 @@ Type: `String`
 
 The path to the translation file to be upload to your OneSky project.
 
+#### options.files
+Type: `Array of Object`
+
+```
+[{
+    ProjectId : String, locale : 'String', file : 'String'
+}, {...}]
```

Comment:
The Grunt standard supports files defined in glob patterns or an array of paths. It does not support an array of objects. While we can force it to interpret our `files` property any way we want, this wouldn't be the following the best practices of Grunt plugins. So, I suggest reducing the `files` property to accepting a string or array or strings, not an array of objects.

If there is need to upload multiple files, each with different locale and projectId options, create a separate tasks in your Gruntfile to handle this.